src/components/dialogs/AccountAuthorizationDialog.vue

```diff
@@ -8,7 +8,7 @@
           retry to access this name request now, or you can exit and return to the dashboard.</p>
       </v-card-text>
 
-      <v-divider></v-divider>
+      <v-divider class="my-0"></v-divider>
```

Comment:
Consistent display across dialogs (and UIs).

Reply:
What does this mean again, is this 'margin top & bottom - 0 '? I need a cheat sheet for these .. :)

Reply:
Yup. See https://vuetifyjs.com/en/styles/spacing/ ! (And also check the other classes for display, flex, etc.)